lambda/health_package/metric-settings.json

```diff
@@ -46,38 +248,410 @@
+    "MetricName": "PutRecords.Bytes",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 759,
+    "Description": "The number of bytes put to the Kinesis stream using the PutRecords operation over the specified time period."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "PutRecords.FailedRecords",
+    "Statistic": "Average",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 0.99,
+    "Description": "The number of records rejected due to internal failures in a PutRecords operation per Kinesis data stream, measured over the specified time period. Occasional internal failures are to be expected and should be retried."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "PutRecords.Latency",
+    "Statistic": "Average",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 2,
+    "Maximum": 60,
+    "Description": "The time taken (msecs) per PutRecords operation, measured over the specified time period."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "PutRecords.Success",
+    "Statistic": "Minimum",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 0.99,
+    "Description": "The number of PutRecords operations where at least one record succeeded, per Kinesis stream, measured over the specified time period."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "PutRecords.SuccessfulRecords",
+    "Statistic": "Minimum",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1.1,
+    "Minimum": 0.99,
+    "Description": "The number of successful records in a PutRecords operation per Kinesis data stream, measured over the specified time period."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "PutRecords.ThrottledRecords",
+    "Statistic": "Maximum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 0.99,
+    "Description": "The number of records rejected due to throttling in a PutRecords operation per Kinesis data stream, measured over the specified time period."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "ReadProvisionedThroughputExceeded",
+    "Statistic": "Minimum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 0.99,
+    "Description": "When the Minimum statistic has a value of 1, all records were throttled for the stream during the specified time period."
+  },
+  {
+    "Namespace": "AWS/Firehose",
```

Comment:
I can only find this one as a Kinesis metric.

Reply:
not sure what happened here, removed.